### ISSUE TITLE
#-1021516755------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1849]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-footprint" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-d36d3865-049b-6661-952e-4a5f2501e7a1">footprint</div>`

<strong>Element Location: </strong>#attr-label-footprint

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1021516755_All page content should be contained by landmarks.png`
